packages/core/content-manager/admin/src/pages/EditView/components/FormLayout.tsx

```diff
@@ -86,7 +98,11 @@ const FormLayout = ({ layout, hasBackground = false }: FormLayoutProps) => {
                         direction="column"
                         alignItems="stretch"
                       >
-                        <InputRenderer {...fieldWithTranslatedLabel} />
+                        {modelContext ? (
+                          <InputRendererWithContext {...fieldWithTranslatedLabel} />
+                        ) : (
+                          <InputRenderer {...fieldWithTranslatedLabel} />
+                        )}
```

Comment:
What's stopping us from always rendering the component that has the context?

For the "root" relation (not in a modal) I guess the context could be initialized with the values from the URL (useDoc stuff)